core/storage/question/gae_models.py

```diff
@@ -152,24 +152,62 @@ class QuestionSkillLinkModel(base_models.BaseModel):
     skill_id = ndb.StringProperty(required=True, indexed=True)
 
     @classmethod
-    def create(
-            cls, question_id, skill_id):
+    def get_model_id(cls, question_id, skill_id):
+        """Returns the model id by combining the questions and skill id.
+
+        Args:
+            question_id: str. The ID of the question.
+            skill_id: str. The ID of the skill to which the question is linked.
+
+        Returns:
+            str. The calculated model id.
+        """
+        return question_id + ':' + skill_id
```

Comment:
'%s:%s' % (question_id, skill_id)

(we generally favour interpolation when doing string manipulation)